integration/setup/config.go

```diff
@@ -144,10 +144,11 @@ func NewConfig() (Config, error) {
 	var helmClient *helmclient.Client
 	{
 		c := helmclient.Config{
-			Logger:          logger,
-			K8sClient:       cpK8sClients.K8sClient(),
-			RestConfig:      host.RestConfig(),
-			TillerNamespace: tillerNamespace,
+			Logger:               logger,
+			K8sClient:            cpK8sClients.K8sClient(),
+			RestConfig:           host.RestConfig(),
+			TillerNamespace:      tillerNamespace,
+			TillerUpgradeEnabled: true,
```

Comment:
What's the purpose of this change? (sorry if the question is stupid :smile:)

Reply:
Oh that's actually not needed. I was testing something, I'll remove it.